### PR TITLE
[sparkoperator]Support configure `enableBatchScheduler`

### PR DESCRIPTION
**Pull request by TommyLike**
#### What this PR does / why we need it:

To support configure `enableBatchScheduler` in spark operator.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

Depends on: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/pull/567

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by TommyLike** [comment]
@liyinan926  please take a look

File: incubator/sparkoperator/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sparkoperator
 description: A Helm chart for Spark on Kubernetes operator
-version: 0.2.5
+version: 0.2.6
```
Comment (Aisuko) [inline incubator/sparkoperator/Chart.yaml]: Please bump the minor version, like `0.3.0`

File: incubator/sparkoperator/values.yaml [hunk 2]
```diff
@@ -26,6 +26,7 @@ metricsEndpoint: "/metrics"
 metricsPrefix: ""
 resyncInterval: 30
 webhookPort: 8080
+batchSchedulerName: ""
```
Comment (Aisuko) [inline incubator/sparkoperator/values.yaml]: Please add the code annotation for `batchSchedulerName `  here can make more sense.

**Review by Aisuko — CHANGES_REQUESTED** [review]
Thanks for your PR, there may some a piece of advice, please have a look, and don't forget solve the conflict first.

**Comment by TommyLike** [comment]
@liyinan926  please take a look

**Pull request merged**